fili-core/src/test/groovy/com/yahoo/bard/systemconfig/ModuleSetupSpec.groovy

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.systemconfig
+
+import com.yahoo.bard.webservice.config.ConfigResourceLoader
+import com.yahoo.bard.webservice.config.SystemConfig
+
+import org.apache.commons.configuration.Configuration
+
+import spock.lang.Specification
+
+abstract class ModuleSetupSpec extends Specification {
+
+    public static final String MODULE_NAME_KEY = "moduleName"
+    public static final String MODULE_RESOURCE_NAME = "/moduleConfig.properties"
+
+    def "Test that SystemConfig loads for this module"() {
+        expect:
+        SystemConfig.properties.getOrDefault("foo", "foo") == "foo"
```

Comment:
It was my first test, it tested that the applicationConfig or some other piece of conflictable-config didn't prevent SystemConfig from working
I'm sure there's a better way to do this.